frontend/src/components/AddCultureDialog.jsx

```diff
@@ -49,7 +49,7 @@ export default function AddCultureDialog({ open, onClose, onSave }) {
 		}
 
 		// Post-condition: If validation passes, call the onSave prop
-		onSave({ name, periodId: parseInt(periodId, 10) });
+		onSave({ name, period_id:periodId,start:1,end:2});
```

Comment:
tf is going on here? Can I get an explanation?

Reply:
In their application, cultures did not have date-range fields. they should, and thus need to be specified. This is reflected in issue #32